core/src/components/datetime/datetime.tsx

```diff
@@ -153,6 +153,16 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() readonly = false;
 
+  /**
+   * Returns if an individual date (day) is enabled or disabled.
+   *
+   * Returns the enabled state of a day given an ISO 8601 date string.
+   * By default, all days are enabled. Developers can use this method
+   * to write custom disabled date logic. When working with the JS date
+   * object, it is recommended to use the UTC functions (i.e. `getUTCDay()`).
```

Comment:
Ended up removing this bit from the documentation here, but if the developer expects October 10th to be disabled, using `.getDay()` with the `Date` object, will disable based on local time instead of universal time. Using `getUTCDay()` will allow them to enforce the day is disabled for all users, regardless of their local time.